src/app/birthday/birthday.service.spec.ts

```diff
@@ -10,7 +10,38 @@ describe('Service: Birthday', () => {
     });
   });
 
-  it('should ...', inject([BirthdayService], (service: BirthdayService) => {
+  it('Service should exist', inject([BirthdayService], (service: BirthdayService) => {
     expect(service).toBeTruthy();
   }));
+
+  describe('getBirthdayFor behavior', () => {
+    it('Should return Date object', fakeAsync(() => {
+      let methodReturnedValue: Date;
+  
+      getService()
+        .getBirthdayFor('Dima')
+        .subscribe(event =>  methodReturnedValue = event);
+  
+      tick(5000);
+  
+      expect(checkIfDateReturned(methodReturnedValue)).toBeFalsy();
```

Comment:
Also you do not have to create custom functions to check for a type of variable.
Instead use jest/jasmine assert framework like `expect.any(Date)` to check if a variable is instance of `Date`:
```ts
expect(methodReturnedValue).toEqual(expect.any(Date));
```